e2e.bats

```diff
@@ -1,33 +1,82 @@
 #!/usr/bin/env bats
 
-@test "reject because name is on deny list" {
-  run kwctl run annotated-policy.wasm -r test_data/pod.json --settings-json '{"denied_names": ["foo", "test-pod"]}'
+@test "Accept resource with labels already set" {
+  run kwctl run --allow-context-aware -r test_data/pod.json \
+	--replay-host-capabilities-interactions test_data/session_replay.yml \
+	--settings-path test_data/settings.json annotated-policy.wasm
 
   # this prints the output when one the checks below fails
   echo "output = ${output}"
 
   # request rejected
   [ "$status" -eq 0 ]
-  [ $(expr "$output" : '.*allowed.*false') -ne 0 ]
-  [ $(expr "$output" : ".*The 'test-pod' name is on the deny list.*") -ne 0 ]
+  [ $(expr "$output" : '.*allowed.*true') -ne 0 ]
+  [ $(expr "$output" : '.*"patchType":"JSONPatch".*') -eq 0 ]
```

Comment:
It would be better to grep for the new label value to check that it was changed.